src/main/java/spoon/MavenLauncher.java

```diff
@@ -166,6 +168,19 @@ public MavenLauncher(String mavenProject, SOURCE_TYPE sourceType, String[] class
 		init(mavenProject, classpath, profileFilter);
 	}
 
+	/**
+	 * Adds an environment variable to the maven invocation.
+	 * <p>
+	 * <br><strong>Note that you need to call {@link #rebuildClasspath()} after calling this method for changes to take effect.</strong>
+	 *
+	 * @param key the name of the environment variable
+	 * @param value its value
+	 * @see #rebuildClasspath() #rebuildClasspath() for changes to take effect
+	 */
+	public void addEnvironmentVariable(String key, String value) {
+		mavenOptions.addEnvironmentVariable(key, value);
+	}
```

Comment:
Oh, actually, I see one more thing that irks me a little bit. Could we rename this to `set`? To me, `add` implies no deduplication, but that makes no sense for environment variables.

```suggestion
	public void setEnvironmentVariable(String key, String value) {
		mavenOptions.setEnvironmentVariable(key, value);
	}
```

Reply:
Yea, I just copied the name of the maven invoker method, but you are right. I adjusted it.